__tests__/app.test.js

```diff
@@ -26,6 +26,29 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
+  it("GET - 200: Should return an array of 'article objects' with the correct properties.", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles).toBeSortedBy("created_at", { descending: true });
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
```

Comment:
1. 35 is redundant as you are using forEach on line 37 (this method only works on arrays and therefore implies the datatype)
2. If you are using a forEach like this (matching generically with .any(String) for example), you need to add another expect statement with the length of the array to make sure that you are receiving all the objects as intended